src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.sql_in

```diff
@@ -1016,7 +1016,7 @@ name                      |
 description               |
 start_training_time       | 2019-12-16 18:54:33.826414
 end_training_time         | 2019-12-16 18:56:19.106321
-madlib_version            | 1.17-dev
+madlib_version            | 1.17
```

Comment:
Maybe these should also say 1.17.0?

Also:  it doesn't show up here in the PR, but in `deep_learning/madlib_keras_fit.sql_in` the versions in the examples still say 1.16